src/editor/Image.js

```diff
@@ -0,0 +1,80 @@
+define(function (require, exports, module) {
+    "use strict";
+
+    var Caman           = require("caman");
+    var FilerFileSystem = require("fileSystemImpl");
+    var FilerUtils       = require("filesystem/impls/filer/FilerUtils");
+    var FileSystemCache = require("filesystem/impls/filer/FileSystemCache");
+    var Path            = require("filesystem/impls/filer/BracketsFiler").Path;
+    var mimeFromExt     = require("filesystem/impls/filer/lib/content").mimeFromExt;
+
+    function initializeFilterButtons(image, imagePath) {
+        var imageMimeType = mimeFromExt(Path.extname(imagePath));
+        var imageDataRegex = /base64,(.+)/;
+        var $saveBtn = $(".btn-image-filter-save");
+        var $resetBtn = $(".btn-image-filter-reset");
+
+        $resetBtn.click(function() {
+            image.reset();
+            $saveBtn.prop("disabled", true);
+            $resetBtn.prop("disabled", true);
+        });
+
+        $saveBtn.click(function() {
+            var imageBase64Data = image.canvas.toDataURL(imageMimeType);
+            var data = FilerUtils.base64ToBuffer(imageDataRegex.exec(imageBase64Data)[1]);
+
+            FilerFileSystem.writeFile(imagePath, data, {encoding: null}, function(err) {
+                if(err) {
+                    console.error("[Bramble] Image with filters failed to save with: ", err);
+                    return;
+                }
+
+                FileSystemCache.refresh(function(err) {
```

Comment:
I tried removing the function (but kept the button disabling) and it seems that it works without it on my machine. I'm not sure if it'll impact other sections of the code. 

@gideonthomas can you confirm if removing it was okay or that it's needed for some other part of the logic?